frontend/src/components/themes/DefaultTheme.scss

```
@@ -71,6 +71,7 @@
   // ----------------
 
   // - base units
+  --base-smaller: 0.75rem; // ~ 12 px
```

```suggestion
// TODO
  --base-smaller: 0.75rem; // ~ 12 px
```

Na tohle se chci podívat. Můžeme mergnout, předělám později, jen prosím přidat `TODO`.